uiServices/src/testSuiteServices.go

```diff
@@ -141,14 +141,34 @@ func putTestSuites(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if !testSuiteWriterXml(testSuite, testSuitePathDir) {
+	if !testSuiteWriterXML(testSuite, testSuitePathDir) {
 		rw.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
 	rw.WriteHeader(http.StatusNoContent)
 }
 
+func deleteTestSuite(rw http.ResponseWriter, req *http.Request) {
+	testSuitePathDir := getTestSuiteHeader(req)
+	testSuiteName := chi.URLParam(req, "testSuiteName")
+
+	ValidateFileNameAndHeader(rw, req, testSuitePathDir, testSuiteName)
+
+	_, err := os.Stat(fmt.Sprintf("%s%s.xml", testSuitePathDir, testSuiteName))
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Error("Test Suite File Not Found: " + testSuitePathDir + testSuiteName)
+			rw.WriteHeader(http.StatusNotFound)
+			return
+		}
+	} else {
```

Comment:
you don't need the ```else```
have a read of https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow to see how we could make this more "idiomatic" Go